pkg/function/backup_data.go

```diff
@@ -112,13 +112,15 @@ func (*backupDataFunc) Exec(ctx context.Context, tp param.TemplateParams, args m
 	if err != nil {
 		return nil, errors.Wrapf(err, "Failed to create Kubernetes client")
 	}
-	backupID, backupTag, err := backupData(ctx, cli, namespace, pod, container, backupArtifactPrefix, includePath, encryptionKey, tp)
+	backupID, backupTag, fileCount, backupSize, err := backupData(ctx, cli, namespace, pod, container, backupArtifactPrefix, includePath, encryptionKey, tp)
 	if err != nil {
 		return nil, errors.Wrapf(err, "Failed to backup data")
 	}
 	output := map[string]interface{}{
-		BackupDataOutputBackupID:  backupID,
-		BackupDataOutputBackupTag: backupTag,
+		BackupDataOutputBackupID:       backupID,
+		BackupDataOutputBackupTag:      backupTag,
+		BackupDataStatsOutputFileCount: fileCount,
+		BackupDataStatsOutputSize:      backupSize,
```

Comment:
@tdmanv @DeepikaDixit - may need your opinion on this. I am using the same key constants defined for BackupStats for the two new fields. Is this appropriate, or should I define new constants for the BackupData version of FileCount and Size?

Reply:
I think this is fine